### ISSUE TITLE
[customcluster] certificate signed by unknown authority when apply cluster and kcp 

### ISSUE THREAD

**Issue by Xieql** [kurator-dev/kurator#341]
**What happened**:

when apply cc-cluster.yaml and cc-kcp.yaml, an erreor occur

**What you expected to happen**:

cluster and kcp resource created

**How to reproduce it (as minimally and precisely as possible)**:

`k apply -f examples/infra/customcluster/cc-cluster.yaml `

`k apply -f examples/infra/customcluster/cc-kcp.yaml `

**Anything else we need to know?**:

here is the log:

Error from server (InternalError): error when creating "cc-cluster.yaml": Internal error occurred: failed calling webhook "default.cluster.cluster.x-k8s.io": failed to call webhook: Post "https://kurator-webhook-service.kurator-system.svc:443/mutate-cluster-x-k8s-io-v1beta1-cluster?timeout=10s": x509: certificate signed by unknown authority

Error from server (InternalError): error when creating "cc-kcp.yaml": Internal error occurred: failed calling webhook "default.kubeadmcontrolplane.controlplane.cluster.x-k8s.io": failed to call webhook: Post "https://kurator-webhook-service.kurator-system.svc:443/mutate-controlplane-cluster-x-k8s-io-v1beta1-kubeadmcontrolplane?timeout=10s": x509: certificate signed by unknown authority

**Environment**:
- Kurator version:
- Others:

**Comment by hzxuzhonghu** [comment]
Does it occur even without https://github.com/kurator-dev/kurator/pull/338 ?

**Comment by Xieql** [comment]
> Does it occur even without #338 ?

yes, i test is in main branch

**Comment by hzxuzhonghu** [comment]
cc @zirain ptal

**Comment by Xieql** [comment]
here is the log from `kubectl logs -l app.kubernetes.io/name=kurator-cluster-operator -n kurator-system --tail=-1`
2023/06/21 09:26:38 http: TLS handshake error from 10.244.0.1:24221: remote error: tls: bad certificate

**Comment by Xieql** [comment]
here is the log when I try add ` --set certManager.enabled=true --set autoGenerateCert=false`

```console
root# helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true  --set certManager.enabled=true --set autoGenerateCert=false
Error: INSTALLATION FAILED: create: failed to create: namespaces "cert-manager" not found
root# kubectl create namespace cert-manager
namespace/cert-manager created
root# helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true  --set certManager.enabled=true --set autoGenerateCert=false
NAME: cert-manager
LAST DEPLOYED: Wed Jun 21 17:59:26 2023
NAMESPACE: cert-manager
STATUS: deployed
REVISION: 1
TEST SUITE: None
NOTES:
cert-manager v1.12.2 has been deployed successfully!

In order to begin issuing certificates, you will need to set up a ClusterIssuer
or Issuer resource (for example, by creating a 'letsencrypt-staging' issuer).

More information on the different types of issuers and how to configure them
can be found in our documentation:

https://cert-manager.io/docs/configuration/

For information on how to configure cert-manager to automatically provision
Certificates for Ingress resources, take a look at the `ingress-shim`
documentation:

https://cert-manager.io/docs/usage/ingress/
root# k apply -f ./../customcluster/
customcluster.infrastructure.cluster.x-k8s.io/cc-customcluster unchanged
Error from server (InternalError): error when creating "../customcluster/cc-cluster.yaml": Internal error occurred: failed calling webhook "default.cluster.cluster.x-k8s.io": failed to call webhook: Post "https://kurator-webhook-service.kurator-system.svc:443/mutate-cluster-x-k8s-io-v1beta1-cluster?timeout=10s": x509: certificate signed by unknown authority
Error from server (BadRequest): error when creating "../customcluster/cc-custommachine.yaml": CustomMachine in version "v1alpha1" cannot be handled as a CustomMachine: strict decoding error: unknown field "spec.masters", unknown field "spec.nodes"
Error from server (InternalError): error when creating "../customcluster/cc-kcp.yaml": Internal error occurred: failed calling webhook "default.kubeadmcontrolplane.controlplane.cluster.x-k8s.io": failed to call webhook: Post "https://kurator-webhook-service.kurator-system.svc:443/mutate-controlplane-cluster-x-k8s-io-v1beta1-kubeadmcontrolplane?timeout=10s": x509: certificate signed by unknown authority
```

**Comment by zirain** [comment]
seems I miss change the service name, should rename `kurator-webhook-serivce` to `kurator-cluster-operator`

**Comment by zirain** [comment]
fixed by https://github.com/kurator-dev/kurator/pull/343, please rebase your branch.

**Comment by Xieql** [comment]
> fixed by #343, please rebase your branch.

ok
